airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py

```diff
@@ -90,10 +90,12 @@ def get_updated_state(self, current_stream_state: MutableMapping[str, Any], late
         Return the latest state by comparing the cursor value in the latest record with the stream's most recent state object
         and returning an updated state object.
         """
-        latest_benchmark = latest_record[self.cursor_field]
-        if current_stream_state.get(self.cursor_field):
-            return {self.cursor_field: max(latest_benchmark, current_stream_state[self.cursor_field])}
-        return {self.cursor_field: latest_benchmark}
+        updated_state = latest_record.get(self.cursor_field, self._replication_start_date.strftime("%Y-%m-%d %H:%M:%S"))
+        stream_state_value = current_stream_state.get(self.cursor_field)
```

Comment:
I think `self._replication_start_date.strftime("%Y-%m-%d %H:%M:%S")` can ba also default value for `current_stream_state`.